salt/modules/gpg.py

```diff
@@ -97,24 +97,25 @@ def _check_gpg():
     Looks to see if gpg binary is present on the system.
```

Comment:
The docstring should now read "Returns the path to the gpg binary", as it is no longer checking anything. For that matter, the function should be renamed to simply ``_gpg()``, and all its references updated.